packages/core/src/indexing/sqlite/sqlite-index-api.ts

```diff
@@ -26,22 +26,39 @@ export class SqliteIndexApi implements DatabaseIndexApi {
     this.insertionOrder = new InsertionOrder(dbConnection)
   }
 
+  public getActiveModelsToIndex(): Array<StreamID> {
```

Comment:
using a Set as the return value will make it more efficient to check for inclusion of a stream.  It will have to be a Set<string> though not a Set<StreamID> as the latter would only consider the StreamID as being included if you provided it the exact same StreamID instance at the same memory address